src/gui/render.cpp

```diff
@@ -373,22 +390,19 @@ static void render_reset(void)
 
 	switch (render.src.pixel_format) {
 	case PixelFormat::Indexed8:
-		render.src.start = (render.src.width * 1) / sizeof(Bitu);
+		render.src_start = (render.src.width * 1) / sizeof(Bitu);
 		break;
 	case PixelFormat::BGR555:
-		render.src.start = (render.src.width * 2) / sizeof(Bitu);
-		gfx_flags        = (gfx_flags & ~GFX_CAN_8);
-		break;
 	case PixelFormat::BGR565:
-		render.src.start = (render.src.width * 2) / sizeof(Bitu);
+		render.src_start = (render.src.width * 2) / sizeof(Bitu);
 		gfx_flags        = (gfx_flags & ~GFX_CAN_8);
 		break;
 	case PixelFormat::BGR888:
-		render.src.start = (render.src.width * 3) / sizeof(Bitu);
+		render.src_start = (render.src.width * 3) / sizeof(Bitu);
 		gfx_flags        = (gfx_flags & ~GFX_CAN_8);
 		break;
 	case PixelFormat::BGRX8888:
-		render.src.start = (render.src.width * 4) / sizeof(Bitu);
+		render.src_start = (render.src.width * 4) / sizeof(Bitu);
```

Comment:
maybe hoist the `sizeof(Bitu)`s out of the switch to a common `constexpr auto src_pixel_bytes = sizeof(uintptr_t)` (or a name like that?)

Reply:
I can do as much, but otherwise I don't wanna touch this existing old code too much now.